istio-cni/templates/daemonset.yaml

```diff
@@ -48,7 +48,7 @@ spec:
         # This container installs the Istio CNI binaries
         # and CNI network config file on each node.
         - name: install-cni
-          image: {{ .Values.cni.hub }}/install-cni:{{ .Values.cni.tag }}
+          image: {{ .Values.cni.hub }}/{{ .values.cni.image }}:{{ .Values.cni.tag }}
```

Comment:
Please add default values ( with the old values ).

Reply:
done